src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
`days` is always >= 3 at this point
```suggestion
  if (days < 7) {
```

Reply:
agree